### PR TITLE
Add workaround for RubyMine debugger issue with datadog

### PR DESCRIPTION
**Pull request by ssrihari**
See issue: https://youtrack.jetbrains.com/issue/RUBY-27489
This is a fix only for rubymine users, but seems worth adding to the codebase instead of having to patch it everytime we use the debugger. We can remove it when the original issue has been addressed.

### CODE REVIEW

**Review by harimohanraj89 — APPROVED** [review]
One question about the DD upgrade. Apart from that, looks neat.

File: config/initializers/00_datadog.rb [hunk 1]
```diff
@@ -16,3 +16,18 @@
 
 require "statsd"
 require "metrics"
+
+# This is a workaround for https://youtrack.jetbrains.com/issue/RUBY-27489
+if defined?(:Debase)
+  if Datadog::VERSION::STRING != "0.50.0"
```
[the remark this answers is not in the document]
Comment (prabhanshuguptagit) [reply config/initializers/00_datadog.rb:22]: Upgrading from 43 -> 50 fixed it for me @ssrihari. What does this fix?

[the remark this answers is not in the document]
Comment (ssrihari) [reply config/initializers/00_datadog.rb:22]: It didn't fix it for me though :/ Let me try again and get back.
